### PR TITLE
Dev/ncrocfer/dagred3

### PR DESCRIPTION
**Pull request by ncrocfer**
screen_director1
screen_director2
screen_director3
screen_director4

### CODE REVIEW

File: director/static/script.js [hunk 1]
```diff
@@ -46,38 +46,55 @@ function getNode(task) {
 }
 
 const router = new VueRouter({
-  routes: [
-    {  name: 'worfklow', path: '/:id' }
+  mode: 'history',
```
Comment (anthonyolea) [inline director/static/script.js:49]: IHMO, history mode should be optional

File: director/templates/index.html [hunk 2]
```diff
@@ -178,33 +259,31 @@
...
+    <script src="https://cdn.jsdelivr.net/npm/axios@0.19.0/dist/axios.min.js" integrity="sha256-S1J4GVHHDMiirir9qsXWc8ZWw74PHHafpsHp5PXtjTs=" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/d3@5.16.0/dist/d3.min.js" integrity="sha256-Xb6SSzhH3wEPC4Vy3W70Lqh9Y3Du/3KxPqI2JHQSpTw=" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/dagre-d3@0.6.4/dist/dagre-d3.min.js" integrity="sha256-dPm4TA8Y9PY5q5mmtWMkRGOCMHJDKy34ZrxdbBGA9cs=" crossorigin="anonymous"></script>
+    {% else %}
+    <script src="%% url_for('user.static', filename='vue.min.js') %%"></script>
+    <script src="%% url_for('user.static', filename='vue-router.min.js') %%"></script>
+    <script src="%% url_for('user.static', filename='vuex.min.js') %%"></script>
+    <script src="%% url_for('user.static', filename='vuetify.min.js') %%"></script>
+    <script src="%% url_for('user.static', filename='moment.min.js') %%"></script>
+    <script src="%% url_for('user.static', filename='axios.min.js') %%"></script>
```
Comment (anthonyolea) [inline director/templates/index.html:285]: As you add `d3.min.js` and `dagre-d3.min.js`, you have to add these libs into the `else` block too

Comment (ncrocfer) [reply director/templates/index.html:286]: Fixed, thanks

File: director/static/script.js (hunk 1 above)
Comment (ncrocfer) [reply director/static/script.js]: Good idea, done

**Pull request merged**